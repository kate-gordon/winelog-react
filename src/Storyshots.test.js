import initStoryshots from '@storybook/addon-storyshots';  

initStoryshots({
    config: ({ configure }) =>
      configure(() => {
        require('./components/Button/Button.stories.js');
      }, module),
  });

