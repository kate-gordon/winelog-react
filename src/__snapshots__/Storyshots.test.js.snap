// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button Black 1`] = `
<button
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "lightgrey",
      "border": "#66c2ff 3px solid",
      "borderRadius": "5px",
      "cursor": "pointer",
      "fontSize": "15px",
      "padding": "10px",
    }
  }
>
  <span
    aria-label="without-milk"
    role="img"
  >
    🏿
  </span>
  <div>
     Please serve 1 cup of coffee 
  </div>
</button>
`;

exports[`Storyshots Button White 1`] = `
<button
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "lightgrey",
      "border": "#66c2ff 3px solid",
      "borderRadius": "5px",
      "cursor": "pointer",
      "fontSize": "15px",
      "padding": "10px",
    }
  }
>
  <span
    aria-label="with-milk"
    role="img"
  >
    🏼
  </span>
  <div>
     Please serve 1 cup of coffee 
  </div>
</button>
`;

exports[`Storyshots CoffeeButton Black 1`] = `
<button
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "lightgrey",
      "border": "#66c2ff 3px solid",
      "borderRadius": "5px",
      "cursor": "pointer",
      "fontSize": "15px",
      "padding": "10px",
    }
  }
>
  <span
    aria-label="without-milk"
    role="img"
  >
    🏿
  </span>
  <div>
     Please serve 1 cup of coffee 
  </div>
</button>
`;

exports[`Storyshots CoffeeButton White 1`] = `
<button
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "lightgrey",
      "border": "#66c2ff 3px solid",
      "borderRadius": "5px",
      "cursor": "pointer",
      "fontSize": "15px",
      "padding": "10px",
    }
  }
>
  <span
    aria-label="with-milk"
    role="img"
  >
    🏼
  </span>
  <div>
     Please serve 1 cup of coffee 
  </div>
</button>
`;
